slurm-cookbook/docker/README.md

```diff
@@ -0,0 +1,66 @@
+# Example: HPC Toolkit Support Slurm and Docker 
+
+## Overview
+
+This example creates a Slurm Cluster and allows Docker files to run.
+All other components required to support the Slurm cluster are minimally created
+as well: VPC; subnetwork; firewall rules; service accounts.
+
+# Install HPC Toolkit
+
+Follow [the instructions here](https://github.com/GoogleCloudPlatform/hpc-toolkit#quickstart) to install the HPC Toolkit.
+In the Hpc Toolkit folder run an install.
+```
+make install
+```
+
+# Run the HPC Toolkit
+
+* Create a GCP Project [as described here](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+ [![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://shell.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/scientific-computing-examples.git)
+
+```
+cd slurm-cookbook/docker
```

Comment:
You never tell them to git close slurm-cookbook

Reply:
Clicking on that button, in theory, clones the repo. Did you give that a try?

I added an alternative to do it manually.

Reply:
OK, I never use Cloud Shell so I didn't click the button. Thanks for adding the alternative.